Sources/Extraction/Filters/HillOrientation.c

```diff
@@ -1,26 +1,183 @@
 #include <stdlib.h>
 #include <assert.h>
+#include <math.h>
+#include <stdio.h>
 
 #include "HillOrientation.h"
+#include "General/Array.h"
 
-HillOrientation HillOrientation_Construct(void)
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
```

Comment:
We need to have a common include for these, this is the second time I have read this code today!